### ISSUE TITLE
How many characters for EURING codes

### ISSUE THREAD

**Issue by peterdesmet** [ices-tools-dev/esas#143]
@nicolasvanermen how many characters are EURING codes? I always assumed 5 characters, with leading zeros?

```
00120
02059
12360
62800
```

Is that correct?

**Comment by peterdesmet** [comment]
Once confirmed, the following codes should be fixed in the ICES DB:

- [ ] Limit to 5, not 6 characters
- [ ] Drop one leading `0` for most records
  - [ ] If this is not done, change 62540 to 062540
- [ ] Add leading zeros for:
  - [ ] 1240 to 01240
  - [ ] 1470 to 01470
  - [ ] 2350 to 02350
  - [ ] 2980 to 02980
  - [ ] 4839 to 04839
  - [ ] 6220 to 06220
  - [ ] 6310 to 06310
  - [ ] 7960 to 07960
  - [ ] 8000 to 08000
  - [ ] 8009 to 08009
  - [ ] 9950 to 09950

**Comment by nicolasvanermen** [comment]
Yes , up until now euring codes always use 5 characters; see [https://app.bto.org/euringcodes/species.jsp](https://app.bto.org/euringcodes/species.jsp)

**Comment by peterdesmet** [comment]
Thanks Nicolas

@cmspinto @Osanna123 can the species list in the ESAS database be updated so that EURING codes are always 5 characters. That way, they are also returned correctly by the API. See https://github.com/ices-tools-dev/esas/issues/143#issuecomment-1825864727 for details.
